s/features/notes/components/notes-indicator/xiome-notes-indicator.ts

```diff
@@ -0,0 +1,34 @@
+
+import {ops} from "../../../../framework/ops.js"
+import {makeNotesModel} from "../../models/notes-model.js"
+import {ComponentWithShare, mixinStyles, html} from "../../../../framework/component.js"
+import bell from "../../../../framework/icons/bell.svg.js"
+import {property} from "../../../../framework/component.js"
+
+import styles from "./xiome-notes-indicator.css.js"
+
+@mixinStyles(styles)
+export class XiomeNotesIndicator extends ComponentWithShare<{
+		notesModel: ReturnType<typeof makeNotesModel>
+	}> {
+	@property({attribute: 'no-icon' })
+	noIcon: boolean = true;
+ 
+	render() {
+		
+		const {state} = this.share.notesModel
+		const stats = ops.value(state.statsOp) ?? {
+			newCount: 0,
+			oldCount: 0,
+		}
+		
+		return html`
+				${stats.newCount === 0 ? html`<span class="bell">${bell}</span>`
```

Comment:
- please refactor this block so there's no repetition (the bell span is repeated here)